server/devpi_server/views.py

```diff
@@ -667,7 +667,9 @@ def index_modify(self):
             json = ixconfig
         kvdict = getkvdict_index(self.xom, json)
         try:
+            oldconfig = dict(stage.ixconfig)
             ixconfig = stage.modify(**kvdict)
+            stage.customizer.on_modified(self.request, oldconfig)
         except InvalidIndexconfig as e:
```

Comment:
Can you catch ``HTTPException`` to let the plugin developer lnow that he should use ``apifatal`` ?

Reply:
This is fixed with "Add on_modified method for stage customizer plugins."